ank/src/cli_commands/apply_manifests.rs

```diff
@@ -96,7 +96,16 @@ pub fn update_request_obj(
 ) -> Result<(), String> {
     for workload_path in paths.iter() {
         let workload_name = &workload_path.parts()[1];
-        let cur_workload_spec = cur_obj.get(workload_path).unwrap().clone();
+        cur_obj
+            .clone()
+            .check_if_provided_path_exists(workload_path)
```

Comment:
In general I like how you solved the problem, but I have two points:
* We should try avoiding the clone here if we can. The problem is that `check_if_provided_path_exists` takes a `&mut`. See the comment there for an idea.
* I needed some time to understand what this does, as the error string which gives the information is inside the method. As the `check` function here is fairly simple, we could also return a boolean and spare ourselves the result and make an if out of it.

Reply:
Done.